aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSyncExtensions.java

```diff
@@ -145,7 +145,40 @@ public String toString() {
         CONFLICT_UNHANDLED("ConflictUnhandled"),
 
         /**
-         * An Unauthorized error will occur if the provided credentials are not authorized for the requested operation.
+         * An internal error occurs when trying to resolve a conflict.
+         * Example: Lambda conflict handler returned a malformed response. Or, cannot invoke Lambda
+         * conflict handler because the supplied resource LambdaConflictHandlerArn is not found.
+         */
+        CONFLICT_ERROR("ConflictError"),
+
+        /**
+         * Max retry attempts were reached for conflict resolution.
+         * Example: Too many concurrent requests on the same object. Before the conflict is resolved,
+         * the object is updated to a new version by another client.
+         */
+        MAX_CONFLICTS("MaxConflicts"),
```

Comment:
Are you tracking retry attempts or number of conflicts? If it's retry attempts, maybe naming it `MaxRetries` would make more sense.

Reply:
I simply listed all of the possible server [errors from AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/conflict-detection-and-sync.html#errors) in case that the customer wants to know :)